src/bot/commands/news.ts

```diff
@@ -2,6 +2,7 @@ import { match } from '../../common/helpers/generic';
 import { TextChannel, User, Message, Emoji, Client, GuildChannel } from 'discord.js';
 
 const linkRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/;
+const directRegex = /^ *(-direct)/
```

Comment:
This regex doesn't work.

Reply:
What do you mean by that? The command is only valid when it's used as first argument (hence the `^`), like this:
```
!news -direct www.google.it Some random comment
```
Hence why I also did the undiscriminated `.replace` call below, since this could only match at most once anyway.

Reply:
I think we would want people to be able to put the parameter any where in the message, some might try to place it at the end and it wouldn't work.